src/service/service_core.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package helium;
+
+enum gateway_staking_mode {
```

Comment:
Can we just move these into individual files like `data_rate.proto` and `staking_mode.proto` to avoid the kitchen sink files syndrome?

Reply:
ev one needs a kitchen sink...but sure...